conformance/README.md

```diff
@@ -31,6 +31,80 @@ This will produce `junit.xml` and `report.html` with the results.
 
 Note: for some registries, you may need to create `OCI_NAMESPACE` ahead of time.
 
+#### Testing registry workflows
+
+The tests are broken down into 4 major categories:
+
+1. Pull
+2. Push
+3. Discovery
+4. Management
+
```

Comment:
suggest using Derek's? naming breakdown and brief description to avoid the question (i edited it a bit):

1. Pull - Highest priority - All OCI registries MUST support pulling OCI container images.
2. Push - Registries need a way to get content to be pulled, but clients can/should be more forgiving here. For example, if needing to fallback after an unsupported endpoint.
3. Content Discovery - Includes tag listing (and possibly search in the future).
4. Content Management - Lowest Priority - Includes tag, blob, and repo deletion. (Note: Many registries may have other ways to accomplish this than the OCI API.)

Reply:
took care of this, thanks